app/DynamicGraphs/GraphGenerator.hs

```diff
@@ -0,0 +1,73 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+
+module DynamicGraphs.GraphGenerator where
+
+import           Data.Graph.Inductive
```

Comment:
The continuous integration check reports an error at this line, because this import is (apparently) redundant, so it should be removed.

You might not see warnings when compiling if this file hasn't been changed for a while. To double-check (and fix) all compiler warnings, you'll want to first run `stack clean`, and then `stack build`. This causes all files to be recompiled, and you should see any warnings appear.